graph/common/Encoding.java

```diff
@@ -52,6 +52,14 @@
     public static final String ROCKS_SCHEMA = "schema";
     public static final int ENCODING_VERSION = 1;
 
+    public interface Partition {
+        short DEFAULT = 0;
+        short VARIABLE_START_EDGE = 1;
+        short FIXED_START_EDGE = 2;
+        short OPTIMISATION_EDGE = 3;
+        short STATISTICS = 4;
+    }
```

Comment:
I think looking at this we should just put the Partition Enum here, and encode three things: `name`, `shortID`, and `fixedLengthStart`.
We should then make a TODO that says the `shortID` should eventually replace the `name` if/when we break backward compatibility again

Reply:
Me and @flyingsilverfin had a verbal discussion and agreed to keep the "partition key" enum in the `Storage` class and the "partition encoding" enum in the `Encoding` class. They will be unified when we refactor the `Encoding` class in the future.

Reply:
The TODO has been added as per the last commit.